resources/user-guides/attach-data.md

```diff
@@ -315,7 +320,7 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### How to attach additional mbox and profile parameters to Target location displayed events
+### How to attach additional Target parameters to Target location displayed events
```

Comment:
Please reword to:
"Attaching additional Target parameters to Target location-displayed events"